sdk/boms/azure-sdk-bom/pom.xml

```diff
@@ -46,184 +46,209 @@
+        <version>11.1.3</version>
       </dependency>
+
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-storage-common</artifactId>
-        <version>12.7.0</version>
+        <artifactId>azure-security-keyvault-certificates</artifactId>
+        <version>4.1.4</version>
       </dependency>
+
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-storage-file-datalake</artifactId>
-        <version>12.1.2</version>
+        <artifactId>azure-security-keyvault-keys</artifactId>
+        <version>4.2.4</version>
       </dependency>
+
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-storage-file-share</artifactId>
-        <version>12.5.0</version>
+        <artifactId>azure-security-keyvault-secrets</artifactId>
+        <version>4.2.4</version>
       </dependency>
+
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-storage-queue</artifactId>
-        <version>12.5.2</version>
+        <artifactId>azure-storage-blob</artifactId>
+        <version>12.10.0</version>
       </dependency>
 
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-ai-textanalytics</artifactId>
-        <version>1.0.0</version>
+        <artifactId>azure-storage-blob-batch</artifactId>
+        <version>12.8.0</version>
       </dependency>
 
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-core-tracing-opentelemetry</artifactId>
-        <version>1.0.0-beta.5</version>
+        <artifactId>azure-storage-blob-cryptography</artifactId>
+        <version>12.10.0</version>
       </dependency>
 
-      <!-- Dependencies used by the Azure SDKs -->
       <dependency>
-        <groupId>com.fasterxml.jackson.core</groupId>
-        <artifactId>jackson-annotations</artifactId>
-        <version>2.10.1</version>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-storage-common</artifactId>
+        <version>12.10.0</version>
       </dependency>
 
       <dependency>
-        <groupId>com.fasterxml.jackson.core</groupId>
-        <artifactId>jackson-core</artifactId>
-        <version>2.10.1</version>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-storage-file-datalake</artifactId>
+        <version>12.4.0</version>
       </dependency>
 
       <dependency>
-        <groupId>com.fasterxml.jackson.core</groupId>
-        <artifactId>jackson-databind</artifactId>
-        <version>2.10.1</version>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-storage-file-share</artifactId>
+        <version>12.8.0</version>
       </dependency>
 
       <dependency>
-        <groupId>com.fasterxml.jackson.dataformat</groupId>
-        <artifactId>jackson-dataformat-xml</artifactId>
-        <version>2.10.1</version>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-storage-internal-avro</artifactId>
```

Comment:
Do we need this as part of the BOM?

Reply:
Possibly not, but I'm including it as it is a library we ship even if it is only an internal dependency